containers/unit_tests/TestDynViewAPI_rank12345.hpp

```diff
@@ -20,4 +20,43 @@ namespace Test {
 TEST(TEST_CATEGORY, dyn_rank_view_api_operator_rank12345) {
   TestDynViewAPI<double, TEST_EXECSPACE>::run_operator_test_rank12345();
 }
+
+template <typename SharedMemorySpace>
+void test_dyn_rank_view_resize() {
+  int n = 1000000;
```

Comment:
I am having trouble reproducing even with the current form even though it matches the failing `MueLu` case.
I can only see the missing fence in the kernel logger so this is more of a best effort for a test case.

Reply:
I can see that we need large enough, Otherwise the kernel is done before we launch. One suggestion for better reproducebility: do the host loop in reverse. Because CUDA does tend to schedule in order.